frontend/src/components/StockDetails/CompanyNews/CompanyNews.js

```diff
@@ -1,74 +1,68 @@
-import React from "react";
-import {
-    Card, CardBody, CardImg,
+import React, { Component } from "react";
+import { 
+    Card, CardBody, CardImg, 
     CardTitle, CardSubtitle, CardText,
     Row, Col, Container
 } from "reactstrap";
-import { companyNews } from "../../../global";
 import { toDate, format } from 'date-fns'
 import "./CompanyNews.css";
-import Prediction from '../Prediction/Prediction'
+import { getCompanyNews } from "../../../service/CompanyNews/CompanyNews";
 
 /**
  * Shows company news.
  * 
  * @return {JSX.Element} Company news.
  */
-class CompanyNews extends React.Component {
-
-    constructor(props){
-        super(props)
+export default class CompanyNews extends Component {
+    constructor(props) {
+        super(props);
         this.state = {
-            companyName: props.companyName
-        }
-    }
-
-    componentDidMount(){
-        //this is where we will call the company news api
+            companyNews: []
+        };
     }
 
-    /**
-     * Converts unix time to date.
-     * 
-     * @param {number} unixTime Unix time.
-     * @return {string} Date in format: Month Day, Year.
-     */
-    convertUnixTimeToDate(unixTime) {
-        const date = toDate(unixTime);
-        return format(date, "MMMM dd, yyyy");
+    componentDidMount() {
+        // Load company news
+        getCompanyNews("AAPL")
```

Comment:
You can use `this.props.companyName` to get the company name from the props here